tests/integration/test_install_basic.py

```diff
@@ -405,8 +405,6 @@ def test_install_venv_project_directory(PipenvInstance):
             prefix="pipenv-", suffix="temp_workon_home"
         ) as workon_home:
             os.environ["WORKON_HOME"] = workon_home
-            if "PIPENV_VENV_IN_PROJECT" in os.environ:
```

Comment:
These tests aren't about this variable and shouldn't impact the test run since it use `with temp_environ()`.